roxie/ccd/ccdstate.cpp

```diff
@@ -2720,6 +2722,13 @@ class CRoxiePackageSetManager : implements IRoxieQueryPackageManagerSet, impleme
                     parallelAggregate = 1;
                 topology->setPropInt("@parallelAggregate", parallelAggregate);
             }
+            else if (stricmp(queryName, "control:pauseEventRecording")==0)
```

Comment:
The presence of startEventRecording and stopEventRecording as two separate requests makes this option supporting both pause and resume feel out of place. It may just be me, though.